packages/view-types/heatmap/src/HeatmapSubscriber.js

```diff
@@ -122,6 +123,10 @@ export function HeatmapSubscriber(props) {
     obsSetsStatus,
   ]);
 
+  const [uint8ObsFeatureMatrix, obsFeatureMatrixExtent] = useUint8ObsFeatureMatrix(
+    { obsIndex, featureIndex, obsFeatureMatrix },
```

Comment:
Don't think this function actually uses these other arguments besides `obsFeatureMatrix`